### DISCUSSION TITLE
Duplicate repos of trending dashboard

### DISCUSSION THREAD

**Discussion by yingfeng** [community/community#82425] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

<img width="987" alt="截屏2023-12-27 20 23 31" src="https://github.com/community/community/assets/7248/6291b4a9-9444-4b28-8081-af703a048740">

**Comment by Iceberry-qdd** [comment]
I also found this problem.

**Comment by servusdei2018** [comment]
Occurring for me as well.
